packages/api-openfin/src/window.ts

```diff
@@ -236,18 +245,20 @@ class Window implements ssf.Window {
 
   getParentWindow() {
     return new Promise((resolve, reject) => {
-      if (this.innerWindow) {
-        let parent = null;
-        if (window.opener) {
-          const win = fin.desktop.Window.wrap(this.innerWindow.uuid, window.opener.name);
-          parent = new Window(null, null, null);
-          parent.innerWindow = win;
+      fin.desktop.InterApplicationBus.publish('ssf-get-parent-window', this.innerWindow.uuid);
+      fin.desktop.InterApplicationBus.subscribe('*' , 'ssf-parent-window', (name) => {
```

Comment:
What happens if `getParentWindow` is invoked more than once? do you get duplicate subscriptions?